index.html

```diff
@@ -14,18 +14,18 @@
    </header>
```

Comment:
there is no need of `p` tag inside `header` tag.

Reply:
Actually P has margin-block of 1 rem by default. header has no margin. i have to give it to header. either way